agent_shared/src/config/mod.rs

```diff
@@ -435,6 +435,7 @@ pub struct EventStoreConfig {
 #[serde(rename_all = "snake_case")]
 pub enum EventStoreType {
     InMemory,
+    MongoDb,
```

Comment:
Do you think this should be the new default or should we leave it at Postgres until we've tested it better?